src/Makefile

```diff
@@ -59,6 +59,8 @@ SRC_C += \
 	main.c \
 	mphalport.c \
 
+# Upgrading?
+# Compare to lib/micropython-microbit-v2/src/codal_port/Makefile
```

Comment:
I might move this to README.md as it could do with mentioning the other semi-duplicated files.